GlobalBehaviorandComponents/templates/components/template-manageapps.html

```diff
@@ -0,0 +1,31 @@
+                    <table class="table table-striped">
+                        <thead>
+                        <tr>
+                            <th>Name</th>
+                            <th>App Type</th>
+                            <th>View</th>
+                            <th>Delete</th>
+                        </tr>
+                        </thead>
+                        <tbody>
+                        {% if applist|length > 0 %}
+                            {% for i in applist %}
```

Comment:
can you make this ` for app in applist `. Easy to read when I got to access the dict. ie `i['label']` is hard to read.

Reply:
@dzadikdev Is `if applist|length > 0` even required? If `applist` is None would not the `for` loop not even happen.